src/providers/MeetingProvider/docs/MeetingManager.stories.mdx

```diff
@@ -232,32 +241,34 @@ To opt-out of event ingestion from the Amazon Chime SDK for JavaScript, provide
 
 ```jsx
 import React from 'react';
-import { NoOpEventReporter, DefaultEventController, DefaultMeetingSessionConfiguration} from 'amazon-chime-sdk-js';
+import {
+  NoOpEventReporter,
+  DefaultEventController,
+  DefaultMeetingSessionConfiguration,
+} from 'amazon-chime-sdk-js';
 import { MeetingProvider } from 'amazon-chime-sdk-component-library-react';
 
 const MeetingForm = () => {
   const meetingManager = useMeetingManager();
-  
+
   const response = await fetch('/my-meetings-endpoint');
```

Comment:
Sorry not your change, can we move this down in the handleJoinMeeting method? We cannot await directly in the component. Sorry I missed this in earlier reviews.

Reply:
Have to move `fetch` and `response.json()` lines inside the handleJoinMeeting async function.
Reference: https://github.com/aws-samples/amazon-chime-sdk/blob/main/apps/meeting/src/containers/MeetingForm/index.tsx#L73-L133

Reply:
Updated.